.eslintrc.js

```diff
@@ -94,5 +94,27 @@ module.exports = {
     "space-infix-ops": [
       "error"
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.mdx", "*.md"],
+      "extends": ["plugin:mdx/recommended"],
+      "settings": {
+        "mdx/code-blocks": true,
+        "mdx/language-mapper": {},
+      },
+      "rules": {
+        "semi": 0,
+        "indent": 0,
+        "no-trailing-spaces": 0,
+        "object-curly-spacing": 0,
+        "space-infix-ops": 0,
+        "no-irregular-whitespace": 0,
+        "no-unused-expressions": 0,
+        "linebreak-style": 0,
+        "react/jsx-no-undef": 0,
+        "no-unused-vars": 0,
+      }
+    }
+  ],
```

Comment:
Any specific reason for these overrides?

Reply:
HI @Nikhil-Ladha,

I am not familiar with the interactions of eslint, vscode, and using multiple plugins.  

I believe the [MDX eslint plugin](https://paulie.dev/posts/2022/08/mdx-2-breaking-changes-and-gatsby-plugin-mdx-v4/#eslint-mdx) to be helpful in creating/updating mdx files to be compatible with MDX v2, but I was encountering an issue where it seemed like rules from other `extended` plugins, were not allowing commits/saves or were reformatting the mdx files on save (since I added the file type to the `checklint` command).

Please let me know if there is an issue with the configuration or any other thoughts on how to have the .mdx files only get linted by the plugin.

Do .mdx files currently get linted or checked for MDX v1 compatibility in the current workflow?

Cheers,
Randy

Reply:
> Do .mdx files currently get linted or checked for MDX v1 compatibility in the current workflow?
Good question, and I doubt that happens as of now. 
While commenting, I didn't notice that the override was for `.md, .mdx` files only. So, this should be fine.

However, we should enable some things in the lint like:
```
"semi": 0,
        "indent": ["error", 2],
        "no-trailing-spaces": 1,
        "object-curly-spacing": 1,
        "space-infix-ops": 0,
        "no-irregular-whitespace": 1,
        "no-unused-expressions": 1,
        "react/jsx-no-undef": 1,
        "no-unused-vars": 1,
```